### DISCUSSION TITLE
How to set  default filter in table filters ?

### DISCUSSION THREAD

**Discussion by sergeynilov** [filamentphp/filament#13724] [Help] [answered]
### Package

Table builder

### Package Version

^3.2-stable

### How can we help you?

In laravel 10 / filamentphp 3 app I ned to set default filter, something like :

```
                Filter::make('filter_viewed_is_viewed')->label('Is viewed')
                    ->query(fn(Builder $query): Builder => $query->where('viewed', ActivityViewedEnum::IS_VIEWED))
                    ->indicator('Is viewed'),

                Filter::make('filter_viewed_is_not_viewed')->label('Is not viewed')
                    ->query(fn(Builder $query): Builder => $query->where('viewed', ActivityViewedEnum::NOT_VIEWED))
                    ->indicator('Is not viewed')
                ->default(ActivityViewedEnum::NOT_VIEWED),

```
I expected option above would be selected as default, but not.

Here
https://filamentphp.com/docs/3.x/tables/getting-started#defining-table-filters
I did not find sucn functionality...

**Comment by micraux** [comment, marked as answer]
Here?
[https://filamentphp.com/docs/3.x/tables/filters/getting-started#applying-the-filter-by-default](https://filamentphp.com/docs/3.x/tables/filters/getting-started#applying-the-filter-by-default)
